app/channels/application_cable/connection.rb

```diff
@@ -9,7 +9,7 @@ def connect
     private
 
     def find_verified_user
-      User.find(id: cookies.signed["user_id"])
```

Comment:
Since we're using `id` as the parameter it's recommend to use the `find` method. Check this out: https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find